python-clusters/create-eks-cluster/cluster.py

```diff
@@ -75,23 +70,21 @@ def start(self):
         # and because 2 different clusters could be concurrently editing the config file
         kube_config_path = os.path.join(os.getcwd(), 'kube_config')
         args = args + ['--kubeconfig', kube_config_path]
+        
+        #we want a fresh kubeconfig file
```

Comment:
```suggestion
        # if a previous kubeconfig exists, it will be merged with the current configuration, possibly keeping unwanted configuration
        # deleting it ensures a coherent configuration for the cluster
```